docs/stately-sky-getting-started.mdx

```diff
@@ -91,20 +91,23 @@ After adding the API key, you’ll need to initialize the actor.
 import { actorFromStately } from '@statelyai/sky';
 
 const actor = actorFromStately({
-  url: 'paste your actor url here',
+  url: 'paste your Sky url here',
   sessionId: 'your session id here',
 });
 ```
 
-The session ID can be any string you want. We recommend using a UUID.
-
 :::tip
-By default Sky is multiplayer. We use the session ID to allow multiple tenants to reference the same running actor instance.
+By default, Sky is multiplayer.
+The session ID is used to shard the multiplayer session.
+
+- Use a shared session ID to allow multiple users to reference the same state.
```

Comment:
```suggestion
- Use a shared session ID to allow multiple users to reference the same actor.
```

Reply:
should this be "reference the same workflow."?